### ISSUE TITLE
student run specify

### ISSUE THREAD

**Issue by ronboger** [ronnyb29/medhacksSite#20]
"Thanks. They need to make that clearer at the top of their home page. It needs to say a "Johns Hopkins University student-run event" instead of just the university name."

**Comment by baileyparker** [comment]
``` javascript
Syntax error: unexpected token `Johns` on line 1:

    "Thanks. They need to make that clearer at the top of their home page. It needs to say a "Johns Hopkins University student-run event" instead of just the university name."
                                                                                              ^
```

Mismatched quote?

**Comment by baileyparker** [comment]
Also, fixed by ff289b92e3891bf273b66cfa9a1bd817165e8d76.
